### ISSUE TITLE
I use python api to build my own network, and find that if the network contains a deconvolution layer, it cannot create a dynamic-size engine. It's too weird！

### ISSUE THREAD

**Issue by 2017ZYS** [NVIDIA/TensorRT#856]
I use python api to build my own network, and find that if the network contains a deconvolution layer, it cannot create a dynamic-size engine. It's too weird！

    de_conv1_w = weights[weight_key_prefix_first+'.6.weight'].detach().cpu().numpy()
    de_conv1_b = weights[weight_key_prefix_first+'.6.bias'].detach().cpu().numpy()
    de_conv1 = network.add_deconvolution(input=de_conv0_relu.get_output(0), num_output_maps=1, kernel_shape=(2, 2), kernel=de_conv1_w,bias=de_conv1_b)
    de_conv1.stride = (2, 2)
    de_conv1.padding = (0, 0)
